test/integration/gpu/itest_slct_manager_with_gpu.py

```diff
@@ -0,0 +1,130 @@
+import textwrap
+from collections.abc import (
+    Iterator,
+)
+from pathlib import Path
+from tempfile import TemporaryDirectory
+from test.integration.ordinary.test_itde_manager import remove_itde
+
+import pytest
+
+from exasol.nb_connector.ai_lab_config import (
+    Accelerator,
+    AILabConfig,
+)
+from exasol.nb_connector.itde_manager import bring_itde_up
+from exasol.nb_connector.language_container_activation import (
+    open_pyexasol_connection_with_lang_definitions,
+)
+from exasol.nb_connector.secret_store import Secrets
+from exasol.nb_connector.slc import ScriptLanguageContainer
+from exasol.nb_connector.slc.script_language_container import CondaPackageDefinition
+
+
+@pytest.fixture(scope="session")
+def flavor(request) -> str:
+    val = "template-Exasol-8-python-3.10-cuda-conda"
+    return val
+
+
+@pytest.fixture(scope="module")
+def working_path() -> Iterator[Path]:
+    with TemporaryDirectory() as d:
+        yield Path(d)
+
+
+@pytest.fixture(scope="module")
+def secrets_file(working_path: Path) -> Path:
+    return working_path / "sample_database.db"
+
+
+@pytest.fixture(scope="module")
+def sample_slc(
+    slc_secrets: Secrets, working_path: Path, flavor: str
+) -> ScriptLanguageContainer:
+    return ScriptLanguageContainer.create(slc_secrets, name=name, flavor=flavor)
+
+
+@pytest.fixture(scope="module")
+def slc_secrets(secrets_file, working_path, flavor) -> Secrets:
+    secrets = Secrets(secrets_file, master_password="abc")
+    secrets.save(AILabConfig.accelerator, Accelerator.nvidia.value)
+    return secrets
+
+
+@pytest.fixture(scope="module")
+def sample_slc(
+    slc_secrets: Secrets, working_path: Path, flavor: str
+) -> ScriptLanguageContainer:
+    return ScriptLanguageContainer.create(slc_secrets, name="sample_gpu", flavor=flavor)
+
+
+@pytest.fixture(scope="module")
+def itde(slc_secrets: Secrets):
```

Comment:
Why? I think, we have something like that already

Reply:
Created a new common fixture: "setup_itde_module"